modular_skyrat/code/modules/cargo/packs/misc.dm

```diff
@@ -0,0 +1,81 @@
+
+//Reminders-
+// If you add something to this list, please group it by type and sort it alphabetically instead of just jamming it in like an animal
+// cost = 700- Minimum cost, or infinite points are possible.
+
+//////////////////////////////////////////////////////////////////////////////
+////////////////////////// Headset Encyption Keys ////////////////////////////
+//////////////////////////////////////////////////////////////////////////////
+
+/datum/supply_pack/misc/encryption/bin_key
```

Comment:
Probably access captain on this too

Reply:
It's only able to be ordered via emagging, and if you're buying something from the black market you probably want to be able to access it without a hassle/lock.

Reply:
mb didn't see hidden